vars/updatePropertyVersion.groovy

```diff
@@ -44,10 +42,8 @@ def call(body) {
 
         sh "git push origin versionUpdate${uid}"
 
-        rs = flow.createPullRequest("[CD] Update release dependencies","${config.project}","versionUpdate${uid}")
-
+        return flow.createPullRequest("[CD] Update release dependencies","${config.project}","versionUpdate${uid}")
```

Comment:
would returning from a closure exit the method? 
Also if you want to make the structure flat, you could use golang's idiom of returning early e.g. 

```groovy
if (!updated) {
  return
}
container(name: 'clients') {
 // ...
 return flow.createPullRequest(...)
}
```

but I am not sure if returning within a closure exits from the caller method (will test this soon) so if it only exists the closure, then this method may not return what `flow.createPull` returned.